### ISSUE TITLE
[tiki100] Remove -startup=2 requirements for graphics

### ISSUE THREAD

**Issue by suborb** [z88dk/z88dk#1003]
If we put all the graphics code in a custom section, we should be able to use PHASE/DEPHASE to assemble it to an address > 32k. Some custom tiki100 code in section code_crt_init in swapgfxbk can then move it to the high memory location.

- [x] -startup=2 requirements removed
- [x] Palette routine fixed
- [x] Gencon supports all 3 screen modes
- [x] Graphics primitives plot in colour

**Comment by suborb** [comment]
I've just tried this out and it doesn't work:

- crt0 defines the SECTION and PHASE
- library file puts routines into that SECTION

Result is that references to the library routines have the address resolved to the unPHASED address

**Comment by suborb** [comment]
I've got something working using a fudge: in classic the code_graphics and bss_graphics can be optionally moved into an overall HIMEM section which is then written as a separate file.

appmake then appends HIMEM and code in the code_crt_init then moves to the right location. An added complication of course is that we also need a HIMEM copy of l_cmp (and probably others)

Now I've got the basics working, I'll create a gencon and get colour plotting working in the same way as other targets (i.e. using the values set by textcolor()/textbackground())
